zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -433,9 +433,12 @@ def cbd_to_nbd(self, req):
         else:
             install_path = cmd.installPath
 
-        port = linux.get_free_port_in_range(10600, 10800)
+        start_port, end_port = linux.parse_port_range(cmd.portRange)
+        port, l = linux.find_free_port_with_locking(start_port, end_port)
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)